### DISCUSSION TITLE
TSA product errors

### DISCUSSION THREAD

**Discussion by Antguz** [davidfrantz/force#203] [Q&A] [answered]
Hi FORCE community,

I am using FORCE for trying to get indexes that are currently in the 3.7.7-develop version (CIre and CCI).  When I try to do it, I got an error message like this one:

![image](https://user-images.githubusercontent.com/7254767/181631151-95dfdbd9-8b35-4d7f-a3e1-d2d3c52f38ef.png)

I am not sure why I got this error message since I can get CIre (with its old feature #152) using version 3.7.6. Perhaps this could be associated with the new harmonic interpolation that I need to add (HARMONIC_MODES = 1, HARMONIC_FIT_RANGE = 2017-01-01 2021-12-31, OUTPUT_NRT = FALSE) even though I just need RBF (INTERPOLATE = RBF). 

I am running FORCE using Singularity in a cluster. Attached you will find my parameter file.

Any guide is always welcome.

Thanks in advance!

AG.

**Comment by davidfrantz** [comment]
Hi @Antguz,

this should not happen obviously. I will take a look. I can't find your parameter file though.

Best,
David

**Reply by Antguz** [reply]
Ups, here it is attached [L3_sentinel2.zip](https://github.com/davidfrantz/force/files/9227194/L3_sentinel2.zip).

As a side note, I guest to you may need to take a look also to the `force-parameter TSA` since the resulting file does not display the arguments `HARMONIC_MODES, HARMONIC_FIT_RANGE, OUTPUT_NRT` 

Thanks again for your help.

AG.

**Comment by davidfrantz** [comment, marked as answer]
Yep, there was a bug.

It is fixed now! And yes, thanks for the hint with ``force-parameter``. I will try to find some time to add the changes soon.

Best,
David
